ping3/command_line.py

```diff
@@ -30,3 +30,6 @@ def main(assigned_args: list = None):
 
     for addr in args.dest_addr:
         ping3.verbose_ping(addr, count=args.count, ttl=args.ttl, timeout=args.timeout, size=args.size, interval=args.interval, interface=args.interface)
+
+if __name__ == "__main__":
```

Comment:
We suppose to use `python3 -m ping3.command_line ...` to run this line, right?

Should we need a `__main__.py` which has the following content?
```py
from . import command_line

if __name__ == "__main__":
    command_line.main()
```